apps/frontend/src/app/Components/CustomNumberInput.tsx

```diff
@@ -68,41 +69,48 @@ export default function CustomNumberInput({
   ...props
 }: CustomNumberInputProps) {
   const { inputProps = {}, ...restProps } = props
+  const {
```

Comment:
component is deprecated, can we move the changes to the one in @genshin-optimizer/ui-common?